.zenodo.json

```diff
@@ -0,0 +1,32 @@
+{
+    "license": "BSD-3-Clause", 
+    "upload_type": "software", 
+    "creators": [
+        {
+            "orcid": "0000-0002-5947-6017", 
+            "affiliation": "Brookhaven National Lab", 
+            "name": "Allan, Daniel B."
+        }, 
+        {
+            "orcid": "0000-0003-4692-608X", 
+            "affiliation": "Brookhaven National Lab", 
+            "name": "Caswell, Thomas"
+        }, 
+        {
+            "orcid": "0000-0003-0746-0547", 
+            "affiliation": "California Polytechnic State University", 
```

Comment:
Since only one affiliation is allowed, please change to `"Pennsylvania State University"`.